api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -169,6 +218,11 @@ private UUID applyForHealthyStartAsPregnantWomanWithNoChildren(LocalDate expecte
         return newDobs;
     }
 
+    private void ageByOneCycle() {
```

Comment:
I'm confused how this can be new to this PR when the method below `ageByFourCycles` isn't new and it calls this method???

Reply:
It's overloaded. May need renaming to be clearer.